src/WormholeSender.sol

```diff
@@ -11,7 +11,7 @@ abstract contract WormholeSender is WormholeBase {
   uint16 public immutable REFUND_CHAIN;
 
   /// @notice The gas limit for cross chain transactions.
-  uint256 constant GAS_LIMIT = 500_000;
+  uint256 public GAS_LIMIT = 200_000;
```

Comment:
var is no longer constant

```suggestion
  uint256 public gasLimit = 200_000;
```

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/94/commits/5fb57cdad6d3b0bce2cf8f63c6c873ef27f761ae